apps/modernization-ui/src/pages/patient/profile/ethnicity/EthnicityForm.tsx

```diff
@@ -47,7 +47,7 @@ export const EthnicityForm = ({ entry, onChanged = () => {}, onCancel = () => {}
         <>
             <Grid row className="flex-justify flex-align-center padding-2">
                 <Grid col={6} className="margin-top-1 label-text">
-                    <label className="required" htmlFor="asOf">
+                    <label style={{ fontWeight: 'bold' }} className="required" htmlFor="asOf">
```

Comment:
Would be nice if there was a `bold` class that could be added instead of specifying `style` tags on each element